app/src/main/java/com/arctouch/codechallenge/api/TmdbApi.kt

```diff
@@ -15,6 +19,14 @@ interface TmdbApi {
         const val API_KEY = "1f54bd990f1cdfb230adb312546d765d"
```

Comment:
Do you think there's a better place to have this keys stored? What you consider as a good practice to store those infos?

Reply:
Sure there is! It can be stored on the gradle configuration, so the build will create the values based on the build variants(So we can have different flavors of the app, using different base url and keys.
If the security of these keys are important here, we can create a native C code just to store the keys.